modulus/models/module.py

```diff
@@ -227,8 +227,8 @@ def save(self, file_name: Union[str, None] = None, verbose: bool = False) -> Non
             if verbose:
                 import git
 
-                repo = git.Repo(search_parent_directories=True)
                 try:
+                    repo = git.Repo(search_parent_directories=True)
```

Comment:
A model could be trained from outside of the git tree cause an exception on the Repo call

Reply:
This makes sense. Tagging @loliverhennigh to verify this is okay.